homeassistant/components/fibaro.py

```diff
@@ -54,6 +55,7 @@
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_URL): cv.url,
         vol.Optional(CONF_PLUGINS, default=False): cv.boolean,
+        vol.Optional(CONF_SECTIONS, default=False): cv.boolean,
```

Comment:
Why does it need to be configurable?

Reply:
if you have a big house, it's nicer to have the section names appended, so you don't end up with two devices with the friendly name being "Bathroom lights". But if you don't have a big house, it's more a drag, because every device will be like "First Floor Bathr..." in most views. Once I added section support, I realized that I don't like it myself, but heard back from people who would appreciate it. So it's a tough call. I don't like too many options either.

Reply:
After unique id feature, users can quite easily change name and entity_id. Do you still think we need to be able to turn sections off?

Reply:
I would have it off by default, but someone with a big house usually would have 100+ devices, renaming those would be a major pain. This option also affects the unique id, to keep the lovelace ui definition file readable as well. I really can see both camps going to a lot of pain and frustration without this flag.

Reply:
Once we'll have area support, I'll map the whole section/room info into areas and then the device name can be the device name. For now, I think it's still the best option however.